docker-compose-dev.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+
+  mongo:
+    image: mongo:3.2.19
+    ports:
+      - "27017:27017"
```

Comment:
@AlvaroVega to run test a mondo db i required, i fought adding a compose for that could be worth